cookbooks/bcpc-hadoop/templates/default/hdp_container-executor.cfg.erb

```diff
@@ -2,4 +2,4 @@ yarn.nodemanager.local-dirs=<%=@mounts.map{ |d| "file:///disk/#{d}/yarn/local" }
 yarn.nodemanager.linux-container-executor.group=yarn
 yarn.nodemanager.log-dirs=<%=@mounts.map{ |d| "file:///disk/#{d}/yarn/logs" }.join(",")%>
 banned.users=hdfs,yarn,mapred,bin      
-min.user.id=1000
+min.user.id=100
```

Comment:
We can pre-create the HBase user to give it a deterministic UID. I'm not particularly concerned about someone running with a "system" UID. Amit, are there particular users you would like to prohibit from running jobs which we could blacklist?